common/lib/xmodule/xmodule/templates/problem/string_response.yaml

```diff
@@ -2,38 +2,37 @@
 metadata:
     display_name: Text Input
     markdown: |
-        A text input problem accepts a line of text from the student, and evaluates the input for correctness based on an expected answer.
-    
-        The answer is correct if it matches every character of the expected answer. This can be a problem with international spelling, dates, or anything where the format of the answer is not clear.
-    
-        >>Which US state has Lansing as its capital?<<
-
-        = Michigan
+        In fill-in-the-blank problems, also known as "text input" problems, learners enter text into a response field. The text can include letters and characters such as punctuation marks. The text that the learner enters must match your specified answer text exactly.
```

Comment:
I would switch these, since we label them text input problems, and other might call them "fill-in-the-blank" problems